Update conformer recipes

VahidooX opened the pull request.

titu1994 commented on nemo/collections/asr/models/ctc_bpe_models.py:
```diff
@@ -77,21 +77,42 @@ def list_available_models(cls) -> Optional[PretrainedModelInfo]:
 
         model = PretrainedModelInfo(
-            pretrained_model_name="stt_en_conformer_ctc_medium",
+            pretrained_model_name="stt_en_conformer_ctc_medium_",
```
Why the extra underscore ?

VahidooX replied:
My mistake! Fixed it.

titu1994 reviewed (approved):
Looks great. I will update the Citrinet configs to match this style, more useful.

fayejf commented on tests/core/test_save_restore.py:
```diff
@@ -134,12 +134,13 @@ def test_EncDecCTCModelBPE(self):
+    # def test_EncDecCTCModelBPE_v2(self):
+    #     # TODO: Switch to using named configs because here we don't really care about weights
+    #     cn = EncDecCTCModelBPE.from_pretrained(model_name="stt_en_conformer_ctc_small")
+    #     self.__test_restore_elsewhere(model=cn, attr_for_eq_check=set(["decoder._feat_in", "decoder._num_classes"]))
```
Why comment it out?

VahidooX replied:
This model is going to get updated with new version. So I updated the version number here, but it is not uploaded on NGC yet. We would enable it when NGC is updated.

fayejf replied:
Got it!

fayejf reviewed (approved):
LGTM!

titu1994 commented on examples/asr/conf/conformer/conformer_ctc_bpe.yaml:
```diff
@@ -1,19 +1,42 @@
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
 # Here are the recommended configs for different variants of Conformer-CTC, other parameters are the same as in this config file.
 # One extra layer (compared to original paper) is added to the medium and large variants to compensate for replacing the LSTM decoder with a linear one.
+#
```
Why are all of these things being duplicated in yaml configs ?? Put them only on NGC cards.

VahidooX replied:
Which parts?
I think none of these parts are on NGC model cards.

titu1994 reviewed (approved):
Info about performance, LM etc belongs in their respective model cards, not in this config. Approved for now, will clean up later.

okuchaiev reviewed (changes requested):
Models are not available. Can not merge it before models are published

For example:

```wget https://api.ngc.nvidia.com/v2/models/nvidia/nemo/stt_en_conformer_ctc_medium_ls/versions/1.0.0/files/stt_en_conformer_ctc_medium_ls.nemo```

titu1994 reviewed (changes requested):
Since the PR needs to be cleaned anyway, please revert info about scores from the config. Place them in their respective model cards.

titu1994 reviewed (dismissed):
Thanks for the cleanup. Lets merge once the checkpoint links work.

okuchaiev commented:
how about we remove links to non-ready checkpoints and merge this?

titu1994 commented:
Agreed. Everything else seems ready to merge

VahidooX commented:
> 
> 
> how about we remove links to non-ready checkpoints and merge this?



> 
> 
> how about we remove links to non-ready checkpoints and merge this?

Dropped the links for new models and reverted to older checkpoints.

okuchaiev reviewed (approved):
lgtm, thanks!
